### PR TITLE
Implements timebase in Oscilloscope UI

### PR DESCRIPTION
**Pull request by akarshan96**
Partly Fixes issue #260 

Changes: 
Timebase seek bar is functional

[apk link ](https://drive.google.com/open?id=0B423XEmWL9vKNlc3YzhMdlhOU1U)
*W1 produces sine wave

Screenshots for the change: 

screenshot_2017-07-06-01-53-59-940_org fossasia pslab

**Here, alternate subsections of 200 data points is reading wrong data**

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/activity/OscilloscopeActivity.java [hunk 1]
```diff
@@ -400,7 +401,6 @@ public void chartInit() {
         mChart.setPinchZoom(true);
         mChart.setScaleYEnabled(false);
         mChart.setBackgroundColor(Color.BLACK);
-        mChart.getDescription().setEnabled(false);
```
Comment (viveksb007) [inline app/src/main/java/org/fossasia/pslab/activity/OscilloscopeActivity.java]: why this?

**Comment by viveksb007** [comment]
@akarshan96 I tested the apk. The amplitude of wave is also quite low.
And **alternate subsections of 200 data points is reading wrong data**, there are packet drops randomly right?

**Comment by akarshan96** [comment]
@viveksb007 Yes, random packet drops are causing this.

**Comment by viveksb007** [comment]
@akarshan96 I don't think its random I am getting distortion at same time instants in all my tests.
screenshot_2017-07-07-16-41-28-125_org fossasia pslab

Even if its random that distorted section should move when triggered but its not.
Could you dig some more into whats the issue here?
Meanwhile I would also look for what can be possible issue.
@jithinbp any views as to why this can happen?

**Comment by viveksb007** [comment]
@akarshan96 any updates on this? or should we merge and you would follow later?

**Comment by akarshan96** [comment]
@viveksb007 merge it, I will follow it later.

**Pull request merged**